### DISCUSSION TITLE
Recent Taskbar Specific Changes in Windows 11...

### DISCUSSION THREAD

**Discussion by Ibuprophen** [Open-Shell/Open-Shell-Menu#1247] [General]
I just figured that this could be more helpful as a discussion rather than a comment within one or more of the issues that may/might be relevant to the following. 

I can't state that this is the issue for everyone, but Personally, I had found that Microsoft had done something recently (in Windows 11) that results in sporadic crashes with `Explorer.exe` and `ExplorerPatcher` either installed and/or being installed. 

I was forced to Uninstall `ExplorerPatcher` manually via command line using `ep_setup /uninstall` that seemed stop the `Explorer.exe` from crashing.

I do know that Microsoft had recently made a change in preventing/removing some taskbar functions that include the taskbar grouping. 

I'm still looking into myself and am hoping to that either @valinet and/or @ge0rdi will be able to figure this one out before I do (LOL!). 

~Ibuprophen

**Comment by ge0rdi** [comment]
You should report `ExplorerPatcher` related crashes to the author of `ExplorerPatcher`.
Microsoft is free to do anything with its OS, third-party SW that hacks into Windows has to adapt.

**Reply by Ibuprophen** [reply]
I planned on reporting this there, but I lost track and forgotten all about it... LOL! 

I had primarily opened this discussion here since it also indirectly affects this specific software as well. 

I know that issues that arise tied (even indirectly) to the Open Shell Software will be popping here for direction and such. 

I noticed some issues opened up that may/might be related to this too. 

Please let me know if you prefer me not to do this any longer as I don't want to step on the developmental team's toes... 

~Ibuprophen
